osdpctl/include/arg_parser.h

```diff
@@ -25,7 +25,7 @@ enum ap_type_e {
 };
 
 struct ap_option {
-	char short_name;
+	signed char short_name;
```

Comment:
Is this needed?

Reply:
yes otherwise the comparison with -1 will not work.
this same issue seems to exist in utils as well(same code).

Reply:
Signed is implied here I think. I'll check this later, thanks.

Reply:
In the C standard it is not specifies if char is signed or unsigned. Some system have unsigned chars as default.

Reply:
Ahh yes! I missed the part where you mentioned you were using other architectures. My bad.